### PR TITLE
Migrate Phase-2 geometry to D98

### PR DESCRIPTION
**Pull request by srimanob**
#### PR description:
This PR move Phase-2 baseline geometry to D98. I also use this PR to clean up old Phase-2 wfs from IB test.

#### PR validation:
Trying to dump the following workflows, and all seems to be fine:
`24834.0,24834.911,24834.103,25061.97,24834.5,24834.9,25034.99,25034.999,24834.21,25034.21,25034.9921,25034.114,24896.0,24900.0`

#### If this PR is a backport please specify the original PR and why you need to backport that PR. If this PR will be backported please specify to which release cycle the backport is meant for:
No need of backport

### CODE REVIEW

File: Configuration/PyReleaseValidation/python/relval_2026.py [hunk 1]
```diff
@@ -16,31 +16,26 @@
 numWFIB = []
 numWFIB.extend([20034.0]) #2026D86
 numWFIB.extend([20834.0]) #2026D88
-numWFIB.extend([21061.97]) #2026D88 premixing stage1 (NuGun+PU)
-numWFIB.extend([20834.5,20834.9,20834.501,20834.502]) #2026D88 pixelTrackingOnly, vector hits, Patatrack local reconstruction on CPU, Patatrack local reconstruction on GPU
-numWFIB.extend([21034.99,21034.999]) #2026D88 premixing combined stage1+stage2 (ttbar+PU200, ttbar+PU50 for PR test)
-numWFIB.extend([20834.21,21034.21,21034.9921]) #2026D88 prodlike, prodlike PU, prodlike premix stage1+stage2
-numWFIB.extend([21034.114]) #2026D88 PU, with 10% OT ineffiency
+numWFIB.extend([20834.501,20834.502]) #2026D88 Patatrack local reconstruction on CPU, Patatrack local reconstruction on GPU (to remove when available in D98)
```
Comment (srimanob) [inline Configuration/PyReleaseValidation/python/relval_2026.py:19]: Do we have .501,.502 versions in D98 yet, @AdrianoDee ?
Thx.

**Comment by srimanob** [comment]
test parameters:
- workflows = 24834.0,24834.911,24834.103,25061.97,24834.5,24834.9,25034.99,25034.999,24834.21,25034.21,25034.9921,25034.114,24896.0,24900.0
- relvals_opt = --what cleanedupgrade,standard,highstats,pileup,generator,extendedgen,production,identity,ged,machine,premix,nano,gpu,2017,2026

**Comment by srimanob** [comment]
@cmsbuild please test

Test all new workflows that will run on IB.

**Comment by srimanob** [comment]
The relvals of 24896.0, and 24900.0, are missing from last submissions with CMSSW_13_2_0_pre1__UPSG_Std_2026D98noPU,

- https://dmytro.web.cern.ch/dmytro/cmsprodmon/workflows.php?campaign=CMSSW_13_2_0_pre1__UPSG_Std_2026D98noPU-1685269946
- https://dmytro.web.cern.ch/dmytro/cmsprodmon/workflows.php?campaign=CMSSW_13_2_0_pre1__UPSG_Std_2026D98noPU-1685264042

Could you please take care for these two workflows, @cms-sw/pdmv-l2 ? Thanks.

**Comment by mmusich** [comment]
@srimanob 

> Can 10% OT inefficiency worrkflow move to D98 (T32) from D88 (T24)?

Technically they can.  If the phase2 tracker group is OK with this change is up to Ernesto to confirm.

**Comment by srimanob** [comment]
> The relvals of 24896.0, and 24900.0, are missing from last submissions with CMSSW_13_2_0_pre1__UPSG_Std_2026D98noPU,
> 
> * https://dmytro.web.cern.ch/dmytro/cmsprodmon/workflows.php?campaign=CMSSW_13_2_0_pre1__UPSG_Std_2026D98noPU-1685269946
> * https://dmytro.web.cern.ch/dmytro/cmsprodmon/workflows.php?campaign=CMSSW_13_2_0_pre1__UPSG_Std_2026D98noPU-1685264042
> 
> Could you please take care for these two workflows, @cms-sw/pdmv-l2 ? Thanks.

Kindly ping @cms-sw/pdmv-l2 for missing samples needed for geometry migration.

**Comment by sunilUIET** [comment]
Hi 

Samples are submitted. Status can be followed here 

https://dmytro.web.cern.ch/dmytro/cmsprodmon/requests.php?campaign=CMSSW_13_2_0_pre1__UPSG_Std_2026D98noPU-1687180512

Sunil

**Comment by srimanob** [comment]
@cmsbuild please test

**Comment by srimanob** [comment]
@cmsbuild please test

**Comment by srimanob** [comment]
+Upgrade

All D98 workflows run fine.

**Comment by srimanob** [comment]
Could you please consider to sign, @cms-sw/pdmv-l2 @cms-sw/geometry-l2 ?
So we can keep testing D98 for every PR. Thanks.

**Comment by sunilUIET** [comment]
+pdmv

**Comment by civanch** [comment]
+1

**Comment by rappoccio** [comment]
+1

**Comment by srimanob** [comment]
Note to @cms-sw/reconstruction-l2 for updating the comparison matrix. Thx.

**Pull request merged**